bin/templates/cordova/lib/prepare.js

```diff
@@ -401,6 +403,44 @@ function mapImageResources(rootDir, subDir, type, resourceName) {
     return pathMap;
 }
 
+
+function updateFileResources(cordovaProject, platformDir) {
+    var files = cordovaProject.projectConfig.getFileResources('android');
+
+    // if there are resource-file elements in config.xml
+    if (files.length === 0) {
+        events.emit('verbose', 'This app does not have additional resource files defined');
+        return;
+    }
+
+    var resourceMap = {};
+    files.forEach(function(res) {
+        var targetPath = path.join(platformDir, res.target);
+        resourceMap[targetPath] = res.src;
+    });
+
+    events.emit('verbose', 'Updating resource files at ' + platformDir);
+    FileUpdater.updatePaths(
+        resourceMap, { rootDir: cordovaProject.root }, logFileOp);
+}
+
+
+function cleanFileResources(projectRoot, projectConfig, platformDir) {
+    var files = projectConfig.getFileResources('android');
+    if (files.length > 0) {
```

Comment:
@dpogue @infil00p  I noticed that files.length is always zero here when I do a `cordova clean` even though I have a `<resource-file>` tag in config.xml.  Is this function useless then effectively?